evap/staff/views.py

```diff
@@ -1571,6 +1581,22 @@ def evaluation_textanswers_update_publish(request):
     return HttpResponseNoContent()
 
 
+@require_POST
+@reviewer_required
+def evaluation_textanswers_update_flag(request):
+    answer = get_object_from_dict_pk_entry_or_logged_40x(TextAnswer, request.POST, "answer_id")
+    assert_textanswer_review_permissions(answer.contribution.evaluation)
+    updated_is_flagged = request.POST.get("is_flagged", None)
+    if updated_is_flagged not in ["true", "false"]:
+        raise SuspiciousOperation
+    updated_is_flagged = updated_is_flagged == "true"
+
+    answer.is_flagged = updated_is_flagged
```

Comment:
```suggestion
    answer.is_flagged = updated_is_flagged == "true"
```
or am I missing something?